### PR TITLE
Add admin action for Sanction actualize (DO-2240)

### PR DESCRIPTION
**Pull request by PyDev777**

### CODE REVIEW

**Comment by IegorSoboliev** [comment]
@PyDev777 let`s discuss logic in a separate meeting with @roman-tiukh ?

File: business_register/admin.py [hunk 1]
```diff
@@ -269,3 +306,30 @@ def has_change_permission(self, request, obj=None):
...
+        )
+        msg = f'Updated successfully! New {msg}.'
+        msg_status = messages.SUCCESS
+    else:
+        msg = f'Already updated! Current {msg}.'
+        msg_status = messages.WARNING
+    o.message_user(req, msg, msg_status)
+
+
+def actualize_noqs(o):
```
Comment (roman-tiukh) [inline business_register/admin.py:331]: what is this?

File: business_register/admin.py [hunk 2]
```diff
@@ -269,3 +306,30 @@ def has_change_permission(self, request, obj=None):
 
     def has_add_permission(self, request, obj=None):
         return self.has_module_permission(request)
+
+
+def actualize_action(o, req):
```
[the remark this answers is not in the document]
Comment (PyDev777) [reply business_register/admin.py:311]: "o" is "self" object

File: business_register/admin.py (hunk 1 above)
Comment (PyDev777) [reply business_register/admin.py:331]: Action can run without items check

File: business_register/admin.py (hunk 2 above)
Comment (roman-tiukh) [inline business_register/admin.py:311]:
```suggestion
def actualize_action(model_admin, request):
```

File: business_register/admin.py [hunk 3]
```diff
@@ -198,6 +225,16 @@ class CompanySanctionAdmin(RegisterModelAdmin):
...
         is_empty_filter('initial_data', 'initial data'),
     )
+    actions = ['actualize', ]
+
+    def actualize(self, request, queryset):
+        actualize_action(self, request)
+    actualize.short_description = 'ACTUALIZE COMPANY SANCTIONS'
+
+    def changelist_view(self, request, extra_context=None):
+        self.req = request
```
Comment (roman-tiukh) [inline business_register/admin.py:235]:
```suggestion
```

File: business_register/admin.py (hunk 1 above)
Comment (roman-tiukh) [inline business_register/admin.py:331]:
```suggestion
def actualize_noqs(model_admin):
```

File: business_register/admin.py [hunk 4]
```diff
@@ -269,3 +306,30 @@ def has_change_permission(self, request, obj=None):
...
+        msg_status = messages.SUCCESS
+    else:
+        msg = f'Already updated! Current {msg}.'
+        msg_status = messages.WARNING
+    o.message_user(req, msg, msg_status)
+
+
+def actualize_noqs(o):
+    if 'action' in o.req.POST and o.req.POST['action'] == 'actualize':
+        post = o.req.POST.copy()
```
Comment (roman-tiukh) [inline business_register/admin.py:333]:
```suggestion
        post = model_admin.request.POST.copy()
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/admin.py (hunk 4 above)
Comment (PyDev777) [reply business_register/admin.py]: Fixed in new commit

File: business_register/admin.py (hunk 1 above)
Comment (PyDev777) [reply business_register/admin.py]: Fixed in new commit

File: business_register/admin.py (hunk 3 above)
Comment (PyDev777) [reply business_register/admin.py]: Fixed in new commit

File: business_register/admin.py (hunk 2 above)
Comment (PyDev777) [reply business_register/admin.py]: Fixed in new commit

[the remark this answers is not in the document]
Comment (PyDev777) [reply business_register/admin.py]: req -> request
Fixed in new commit

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**